package.json

```diff
@@ -30,5 +30,8 @@
         "globalSetup": "jest-environment-puppeteer/setup",
         "globalTeardown": "jest-environment-puppeteer/teardown",
         "testEnvironment": "jest-environment-puppeteer"
+    },
+    "dependencies": {
+        "@types/bootstrap": "^5.1.12"
     }
```

Comment:
Please move this into `devDependencies`, making sure to keep them sorted

Reply:
Additionally, please rerun `npm install` to update `package-lock.json` afterwards